roles/keystone-defaults/defaults/main.yml

```diff
@@ -83,6 +83,7 @@ keystone:
     sensu_checks:
       check_keystone_api:
         criticality: 'critical'
+    cron_bin_path:  "/sbin:/usr/sbin:{{ (os == 'rhel' ) | ternary('/usr/bin/', '/usr/local/bin/') }}"
```

Comment:
agree.. would it make more sense to have this right in template rather than defining variable? it is not used any where else.